lib/commands/network.js

```diff
@@ -0,0 +1,14 @@
+
+let extensions = {}, commands = {};
+
+commands.setWifiState = async function(wifi) {
+  return await this.uiautomator2.jwproxy.command('/appium/device/toggle_wifi', 'POST', {wifi});
```

Comment:
Seems network connection API is not stable, users reported issue https://github.com/appium/appium/issues/6615